python/pypto/ir/op/tensor_ops.py

```
@@ -682,6 +682,25 @@ def expands(target: Expr, scalar: int | float | Expr, span: Span | None = None)
     return _ir_core.create_op_call("tensor.expands", [target, scalar_expr], {}, actual_span)
 
 
+def expand_clone(
+    input: Expr,
```

Ruff (0.15.10)

[error] Function argument `input` is shadowing a Python builtin

(A002)